src/analysis_and_optimization/Mir_utils.ml

```diff
@@ -6,34 +6,33 @@ open Middle.Expr
 open Dataflow_types
 
 let rec fold_expr ~take_expr ~(init : 'c) (expr : Expr.Typed.t) : 'c =
-  Expr.Fixed.Pattern.fold_left
-    ~f:(fun a e -> fold_expr ~take_expr ~init:(take_expr a e) e)
-    ~init expr.pattern
+  Expr.Pattern.fold
+    (fun a e -> fold_expr ~take_expr ~init:(take_expr a e) e)
+    init expr.pattern
 
 let fold_stmts ~take_expr ~take_stmt ~(init : 'c)
     (stmts : Stmt.Located.t List.t) : 'c =
   (* let rec fold_expr (state : 'c) (expr : Expr.t) =
-   *   Expr.Fixed.Pattern.fold_left
+   *   Expr.Pattern.fold_left
```

Comment:
`fold_left` doesn't exist anymore, but this code has been commented out for five years, I think it's time to remove it entirely.